data/models/1x-1x_DM600_LUDVAE.json

```diff
@@ -0,0 +1,36 @@
+{
+    "name": "DM600_LUDVAE",
+    "author": "musl",
+    "license": "CC0-1.0",
+    "tags": [
+        "anime",
+        "cartoon"
+    ],
+    "description": "Create realistic noise and compression artifacts on images. This is meant to synthesize paired dataset for SISR training.",
+    "date": "2023-04-20",
+    "architecture": "ludvae",
+    "size": [],
```

Comment:
Did the local website generate this or did you write the JSON by hand?

Reply:
Yup, it's bugged. I just tested it. Sorry about that. 

Please replace the empty `size` array with `null` by hand in your JSON files.